### PR TITLE
remove rounding, change precision to minute

### PR DESCRIPTION
**Pull request by ngong**
ping @iZettle/horizons

### CODE REVIEW

**Comment by nzroller** [comment]
Awesome man. Merge my PR and the tests might pass :-)

File: dropwizard-metrics-influxdb/src/main/java/com/izettle/metrics/dw/InfluxDbReporterFactory.java [hunk 1]
```diff
@@ -104,6 +104,9 @@
     @NotNull
     private String auth = "";
 
+    @NotNull
+    private TimeUnit timePrecision = TimeUnit.MINUTES;
```
Comment (nzroller) [inline dropwizard-metrics-influxdb/src/main/java/com/izettle/metrics/dw/InfluxDbReporterFactory.java]: @ngong can you change to simply `precision` and add a getter and setter like the other fields? That way it is configurable.

**Comment by ngong** [comment]
@nzroller fixed

**Comment by nzroller** [comment]
Thanks for fixing my weird intepretation of precision-by-rounding :-)

**Pull request merged**